lib/aggregate/container.rb

```diff
@@ -8,9 +8,6 @@ module Container
     include ActiveSupport::Callbacks
     include Aggregate::AggregateStore
 
-    if ActiveRecord::VERSION::MAJOR > 4
```

Comment:
This is surprising that this just went away...

Seems like this is necessary still?